apps/masterbots.ai/components/routes/chat/chat-panel.tsx

```
@@ -65,15 +67,15 @@ export function ChatPanel({
           <PromptForm
-            onSubmit={async value => {
+            onSubmit={async (value) => {
               await append({
                 id,
                 content: value,
-                role: 'user'
-              })
+                role: "user",
+              });
             }}
             disabled={!Boolean(chatbot)}
```

Biome

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)